<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipboardWindow</class>
 <widget class="QMainWindow" name="ClipboardWindow">
  <property name="windowTitle">
   <string>Clipboard</string>
  </property>
  <widget class="QWidget" name="mainWidget">
   <layout class="QGridLayout" name="mainLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QListWidget" name="fileList">
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="emptyInfo">
      <property name="visible">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Clipboard is empty</string>
      </property>
      <property name="alignment">
       <enum>Qt::AlignHCenter | Qt::AlignVCenter</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="OperationsProgressIndicator" name="progressIndicator">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="windowMenu">
    <addaction name="selectAction"/>
    <addaction name="deselectAction"/>
    <addaction name="unclipAction"/>
    <addaction name="clearAction"/>
    <addaction name="copyAction"/>
    <addaction name="moveAction"/>
    <addaction name="linkAction"/>
    <addaction name="removeAction"/>
   </widget>
   <addaction name="windowMenu"/>
  </widget>
  <action name="selectAction">
   <property name="text">
    <string>Select all</string>
   </property>
  </action>
  <action name="deselectAction">
   <property name="text">
    <string>Select none</string>
   </property>
  </action>
  <action name="unclipAction">
   <property name="text">
    <string>Unclip deselected</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="moveAction">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="linkAction">
   <property name="text">
    <string>Link</string>
   </property>
  </action>
  <action name="removeAction">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
